packages/tc-ui/src/primitives/simple-relationship/lib/view-relationship.jsx

```diff
@@ -0,0 +1,61 @@
+const React = require('react');
+const { getType } = require('@financial-times/tc-schema-sdk');
+const { LinkToRecord } = require('../../../lib/components/structure');
+
+let RelationshipAnnotator;
```

Comment:
This reference scares me a bit... I assume a record may have different types of relationship so this may be toggled multiple times to render a page. If rendering ever becomes async - or if async logic added between the `setRelationshipAnnotator()` call and rendering - this could cause some wild problems.

Reply:
Can you add a comment to the 'do tc-ui properly' issue. Would be good to use that as a place to collect things to address. For now though, this is completely server side rendered and the risk is low